components/common/Notification.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <NeoMessage
+    :duration="duration"
+    :title="title"
+    :variant="variant"
+    auto-close
+    show-progress-bar
+    @close="emit('close')">
+    <div class="flex gap-2 flex-col">
+      <p class="text-k-grey text-sm">{{ message }}</p>
+
+      <a
+        v-if="action"
+        v-safe-href="action.url"
+        class="text-[16px] !text-text-color"
+        target="_blank">
+        {{ action.label }}
+      </a>
+    </div>
+  </NeoMessage>
+</template>
+
+<script lang="ts" setup>
+import { NeoMessage, NeoMessageVariant } from '@kodadot1/brick'
+
+type NotificationAction = { label: string; url: string }
+
+const emit = defineEmits(['close'])
+withDefaults(
+  defineProps<{
+    title?: string
```

Comment:
title, message, and action need default value
https://eslint.vuejs.org/rules/require-default-prop.html

This is what I got in my editor. Did you get that also?

<img width="1217" alt="Screenshot 2024-04-20 at 21 59 49" src="https://github.com/kodadot/nft-gallery/assets/734428/fc606b63-36c8-4938-a975-df5a98939ee4">

Reply:
not atm bcz I'm trying [zed](https://github.com/zed-industries/zed) but I'm aware of that 

updated `title`and `message` , what would be the proper default value for `action` ? make it `action?: NotificationAction | null` and default `null`?  not really doing much going from undefined to null and both are nullish

Reply:
`undefined` ok I guess

Reply:
updated